adventure/location.py

```diff
@@ -0,0 +1,4 @@
+
+class Location(object):
```

Comment:
Thanks!  I enjoy writing docstrings and talking to myself on github!  I made the change.